snackbar/snackbar/src/snackbar/Snack.java

```diff
@@ -0,0 +1,73 @@
+package snackbar;
+
+public class Snack {
+    // fields
+    private static int maxId = 0;
+    public int id;
+    public String name;
+    public int quantity;
+    public double cost;
+    public int VID;
```

Comment:
carefull of theses capital letters usually capitals are reserved for constructors, this might make bugs in your code down the line
```suggestion
    public int vID;
```